assets/js/util/date-range.js

```diff
@@ -52,7 +53,7 @@ export function getCurrentDateRange() {
  * @return {string} the date range slug.
  */
 export function getCurrentDateRangeSlug() {
-	return applyFilters( 'googlesitekit.dateRange', 'last-28-days' );
+	return Data.select( 'core/user' ).getDateRange();
```

Comment:
Please use the imported `STORE_NAME as CORE_USER` here instead for consistency.

Reply:
Updated.